src/frontend/js/components/Modal/index.tsx

```diff
@@ -43,6 +66,23 @@ export const Modal = ({
       overlayClassName={mergeClasses({ base: 'modal__overlay', classes: overlayClassName })}
       {...props}
     >
+      <div className={headerClasses.join(' ')}>
+        <div>{title}</div>
```

Comment:
Same this inline content, I think a `p` tag is more appropriate in this case.

Reply:
What do you think about using a `h2` tag ? IMO it has more to do with a title than a paragraph

Reply:
Hmmm... if we consider modal content relevant for the document yep ˋh2` is the right heading level furthermore it allows us use lower heading tags within modal content. 

So let's do it ! 👍

Reply:
All righty :)